gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -219,24 +219,26 @@ <h3 class="mdc-layout-grid__cell--span-3">
               <dd>
                 <div class="mdc-layout-grid__inner events">
                   {% for event in range.events -%}
-                    <div class="mdc-layout-grid__cell--span-3">
+                    <div class="mdc-layout-grid__cell--span-2">
```

Comment:
Are the layout width changes intentional? I do prefer to keep the width the same here if possible.

Reply:
@another-rex Ah, yesterday, I had a afternoon brain and tried a few options, but none worked. Just re-tried and now it's been fixed.